### PR TITLE
Fix NUMBA_OPT and debug flag

### PR DESCRIPTION
**Pull request by 1e-to**
Fix NUMBA_OPT and debug flag

### CODE REVIEW

File: docs/user_guides/debugging/debugging_environment.rst [hunk 1]
```diff
@@ -33,12 +33,11 @@ You will also need to create and activate conda environment with installed `numb
 Activate environment variables
 ------------------------------
 
-You need to set the following variables for debugging:
+You need to set the following variable for debugging:
```
Comment (akharche) [inline docs/user_guides/debugging/debugging_environment.rst:36]: It's a recommendation. May be add something like:
Debugging on "no optimization" level is more stable. Local variable are not optimized out.

And provide link to Numba: http://numba.pydata.org/numba-doc/latest/user/troubleshoot.html#debugging-jit-compiled-code-with-gdb

File: docs/user_guides/debugging/debugging_environment.rst [hunk 2]
```diff
@@ -33,12 +33,11 @@ You will also need to create and activate conda environment with installed `numb
...
 
 .. code-block:: bash
 
-    export NUMBA_OPT=1
-    export NUMBA_DPPY_DEBUGINFO=1
+    export NUMBA_OPT=0
```
Comment (PokhodenkoSA) [inline docs/user_guides/debugging/debugging_environment.rst:40]: Please, write some words about NUMBA_DPPY_DEBUGINFO. You can see on numba docs.

Comment (PokhodenkoSA) [reply docs/user_guides/debugging/debugging_environment.rst:40]: It is not necessary to recommend to set it each time but user should know what is this variable for.

**Comment by 1e-to** [comment]
@PokhodenkoSA , @akharche  please check

**Review by akharche — APPROVED** [review]

File: docs/user_guides/debugging/debugging_environment.rst [hunk 3]
```diff
@@ -33,12 +33,21 @@ You will also need to create and activate conda environment with installed `numb
...
+Debugging on "no optimization" level is more stable. Local variable are not optimized out.
+You need to set the following variable for debugging:
 
 .. code-block:: bash
 
-    export NUMBA_OPT=1
-    export NUMBA_DPPY_DEBUGINFO=1
+    export NUMBA_OPT=0
+
+You can use ``NUMBA_DPPY_DEBUGINFO`` instead of ``debug`` option.
```
Comment (PokhodenkoSA) [inline docs/user_guides/debugging/debugging_environment.rst:43]: I think it is not clear what `debug` option do you mean.

**Pull request merged**